src/components/races/RaceImportModal.tsx

```
@@ -219,11 +231,31 @@ export default function RaceImportModal({ isOpen, onClose, onSuccess }: RaceImpo
 
   const parseCSVFile = useCallback(async (file: File): Promise<ParsedRaceData[]> => {
     return new Promise((resolve, reject) => {
+      console.log('[RaceImport] Starting CSV parse for:', file.name)
       Papa.parse(file, {
         header: true,
         skipEmptyLines: true,
+        error: (error: Error) => {
+          console.error('[RaceImport] Papa.parse failed with error:', error)
+          reject(new Error(`CSV parsing engine failed: ${error.message}`))
+        },
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)